tests/integration/mixer/policy/ratelimiting_test.go

```diff
@@ -280,34 +275,27 @@ func deleteComponentsOrFail(t *testing.T, ctx resource.Context, g galley.Instanc
 		bookinfo.NetworkingVirtualServiceAllV1.LoadWithNamespaceOrFail(t, bookinfoNs.Name()))
 }
 
-func setupConfigOrFail(t *testing.T, con, bookInfoNameSpaceStr, destinationService, quotaSpecCon, quotaRuleCon string,
-	red redis.Instance, g galley.Instance, ctx resource.Context) {
-	con = strings.Replace(con, "<redis_namespace>", red.GetRedisNamespace(), -1)
-	quotaSpecCon = strings.Replace(quotaSpecCon, "<bookinfo_namespace>", bookInfoNameSpaceStr, -1)
-	quotaRuleCon = strings.Replace(quotaRuleCon, "<destination_service>", destinationService, -1)
+func setupConfigOrFail(t *testing.T, config bookinfo.ConfigFile, bookInfoNameSpaceStr string,
+	red redis.Instance, g galley.Instance, ctx resource.Context) string {
+	p := path.Join(env.BookInfoRoot, string(config))
+	con, err := test.ReadConfigFile(p)
```

Comment:
The `test.ReadConfigFile` method is going away soon ... use `file.AsStringOrFail`.

Reply:
Done.